tools/add_colab_instruction.py

```diff
@@ -2,14 +2,21 @@
 import glob, json
 
 build_script = "!curl -O https://raw.githubusercontent.com/aws-samples/d2l-java/master/tools/colab_build.sh && bash colab_build.sh"
+fix_gpu_script = "!curl -O https://raw.githubusercontent.com/aws-samples/d2l-java/01c3fcdcae62a13f363a04b87ef6ff08e64a2ebe/tools/fix-colab-gpu.sh && bash fix-colab-gpu.sh"
 script_constructor = {"cell_type": "code", "metadata": {}, "outputs": [], "source": [build_script], "execution_count": None}
+fix_gpu_constructor = {"cell_type": "code", "metadata": {}, "outputs": [], "source": [fix_gpu_script], "execution_count": None}
 
 md = [
     "## Prepare Java Kernel for Google Colab\n",
     "Since Java is not natively supported by Colab, we need to run the following code to enable Java kernel on Colab.\n",
     "\n",
     "1. Run the cell bellow (click it and press Shift+Enter),\n",
-    "2. then refresh the page (press F5) right after that.\n",
+    "2. (If training on CPU, skip this step) If you want to use the GPU with MXNet in DJL 0.10.0, we need CUDA 10.1 or CUDA 10.2.\n"
+    "Since Colab supports CUDA 10.1, we will have to follow some steps to setup the environment.\n"
+    "Refresh the page (press F5) and stay at Python runtime on GPU. Run the file fix-colab-gpu script.\n"
```

Comment:
There should be a command like `wget -q -O - http://raw.github/awslabs/djl/... | bash` that can be run directly to download and run the file

Reply:
I did add variable fix_gpu_script which uses curl to get the bash file and runs it. It's similar to how we are setting up Java kernel. Like this:

`!curl -O https://raw.githubusercontent.com/aws-samples/d2l-java/01c3fcdcae62a13f363a04b87ef6ff08e64a2ebe/tools/fix-colab-gpu.sh && bash fix-colab-gpu.sh`